### DISCUSSION TITLE
Extend RJSF to a new design system.

### DISCUSSION THREAD

**Discussion by AkshitAggarwal** [rjsf-team/react-jsonschema-form#3439] [Q&A]
What's the right way of extending the functionality of the RJSF tool to a new design system, we have an internal design system and we could really benefit if we had RJSF render forms based on this system's components. I'm currently taking some inspiration from how it's extended for Material UI and other systems using the rjsf/core and rjsf/utility package.

**Comment by leetwito** [comment]
RJSF doesn't have a headless mode, so you can't bring your own design system.
